internal-api/src/main/java/datadog/trace/bootstrap/instrumentation/api/AssortedConstants.java

```diff
@@ -0,0 +1,6 @@
+package datadog.trace.bootstrap.instrumentation.api;
+
+public class AssortedConstants {
```

Comment:
```suggestion
public class WriterConstants {
```
I am afraid that in the future this class will be the "potpourri" of various constants that we don't know where to put....